vector/src/main/java/im/vector/app/features/form/FormEditTextItem.kt

```diff
@@ -76,8 +75,13 @@ abstract class FormEditTextItem : VectorEpoxyModel<FormEditTextItem.Holder>() {
         holder.textInputLayout.error = errorMessage
         holder.textInputLayout.endIconMode = endIconMode ?: TextInputLayout.END_ICON_NONE
 
-        // Update only if text is different and value is not null
-        holder.textInputEditText.setTextSafe(value)
```

Comment:
I would have keep the extension, and add `holder.view` as a parameter to avoid repeating yourself

Reply:
Done